tools/geninterop/geninterop.py

```diff
@@ -351,6 +350,10 @@ def main():
     gen_interop_head(writer)
 
     gen_heap_type_members(ast_parser, writer)
+
+    ver_define = "PYTHON{0}{1}".format(PY_MAJOR, PY_MINOR)
+    writer.append(code="#if " + ver_define)
```

Comment:
Why are these still behind a compile-time flag? Do you plan to change that?

Reply:
Yeah, I am approaching this gradually.

Reply:
I checked this out, and the rest of the conditional stuff came from 1b466df0. Since it was only used to determine if methods in Python.Runtime itself implement slots (which we can do on our own), I took a liberty to mostly revert it in this PR. Please, review the last commit "revert slot-related parts of 1b466df to avoid conditional compilation"